### ISSUE TITLE
[TA] Comments from TA

### ISSUE THREAD

**Issue by seongil-wi** [KAIST-IS521/2018s-presentation-team3#9]
Hi, I am the TA of team 3.
I am monitoring your work progress. It seems good to me to starting well with your good `Milestone`.
There are some comments from TA.

1. Link your email address with Git. 
I saw some commits are broken because they are not linked to a user. 
Refer this [link](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/) for fix it.

2. Create and use `Labels` in issues.
It is very useful for categorizing or organizing issues and pull requests.
Refer this [link](https://help.github.com/articles/about-labels/) for improving your `Labels`.

3.  When you rename and commit some file, just add `deleted` file and renamed file which are in `Untracked` state to the `Committed` state.
Then, you can see that file are exist as `renamed` file when you run `git status`.

(+) Nits: In [README.md](https://github.com/KAIST-IS521/2018s-presentation-team3/blob/master/README.md) team member, there are some broken space.

I hope some comments could be helpful to your works!

Thanks,
Seongil Wi

**Comment by 2dvorak** [comment]
Fixed broken commit link problem by adding locally configured email address to my Github profile. (KAIST email)
For the file renaming problem, I didn't know that I should also ```add``` the deleted file. I used ```add``` for the renamed file then ```rm``` to deleted file to commit renaming, but it seems ```add``` both file is better as git will notice the renaming operation.
Thanks for your advice. Will try to make sure other team members won't have the same problem.
